docs/REST API/rest-api.md

```diff
@@ -3877,26 +3877,26 @@ These API endpoints are used to automate MDM features in Fleet. Read more about
 - [Delete an EULA file](#delete-an-eula-file)
 - [Download an EULA file](#download-an-eula-file)
 
-### Add custom macOS setting (configuration profile)
+### Add custom setting (configuration profile)
 
-Add a configuration profile to enforce custom settings on macOS hosts.
+Add a configuration profile to enforce custom settings on macOS and Windows hosts.
 
-`POST /api/v1/fleet/mdm/apple/profiles`
+`POST /api/v1/fleet/mdm/profiles`
 
 #### Parameters
 
-| Name                      | Type     | In   | Description                                                               |
-| ------------------------- | -------- | ---- | ------------------------------------------------------------------------- |
-| profile                   | file     | form | **Required**. The mobileconfig file containing the profile.               |
-| team_id                   | string   | form | _Available in Fleet Premium_ The team id for the profile. If specified, the profile is applied to only hosts that are assigned to the specified team. If not specified, the profile is applied to only to hosts that are not assigned to any team. |
+| Name                      | Type     | In   | Description                                                                                                   |
+| ------------------------- | -------- | ---- | ------------------------------------------------------------------------------------------------------------- |
+| profile                   | file     | form | **Required.** The .mobileconfig or .xml file containing the profile.                                          |
+| team_id                   | string   | form | _Available in Fleet Premium_ The team id for the profile. If specified, the profile is applied to only hosts that are assigned to the specified team. If not specified, the profile is applied to only to hosts that are not assigned to any team.                         |
 
 #### Example
 
-Add a new configuration profile to be applied to macOS hosts enrolled to Fleet's MDM that are
+Add a new configuration profile to be applied to macOS and Windows hosts enrolled to Fleet's MDM that are
```

Comment:
```suggestion
Add a new configuration profile to be applied to a macOS host
```

Looks like the example is just for macOS? See the `.mobileconfig` in the example.